front/app/utils/participationMethodUtils.tsx

```diff
@@ -166,7 +177,7 @@ const budgetingConfig: ParticipationMethodConfig = {
       }
     }
   },
-  getFormTitle: (project, phases) => {
+  getFormTitle: (project, phases, _phaseFromUrl) => {
```

Comment:
Curious, why are we using the underscore in some places?

Reply:
The linter suggested it for if the variables are there but not being used. Do you have a different suggestion though? I wasn't quite sure what was best practice :)

Reply:
Normally, we shouldn't have them if they are not used but there might be a case when that might stretched. This looks like it is used on line 195.

Reply:
Ah true - I'll remove it from 195 and use just "phaseFromUrl" as the parameter name for that getFormTitle function. Thanks for catching it :)

Reply:
There are some other places in the PR where they used but I didn't leave comments on all of them. We can change them there aswell

Reply:
Additionally, for the cases where we are want to avoid variables that are not used, we can use an object type and define optional types. We can then only pull out what we need from the type.

```
type ExampleType = {
  project?: IProjectData,
  _ideaId?: string,
  _idea?: IIdea,
  phaseId?: string
}
```
We wouldn't need to pass optional parameters in the object. That we would avoid them entirely

Reply:
Oh yes that's right.. I implemented a type like that yesterday for these functions (we had the same comment on my last PR)! I'm not sure what happened - I must have forgot to push the commit or reset my local work by accident. I'll do this for both methods we have.